package.json

```diff
@@ -53,12 +53,11 @@
     "exports": false
   },
   "scripts": {
-    "prepare": "concurrently -n \"lint,compile,test\" -c \"blue,yellow,green\" \"yarn lint\" \"yarn compile\" \"yarn test\" --kill-others-on-fail",
+    "prepare": "concurrently -n \"lint,compile,test\" -c \"blue,yellow,green\" \"yarn lint\" \"yarn compile\" \"yarn test:cover\" --kill-others-on-fail",
     "lint": "yarn run lint:css && yarn run lint:js",
     "lint:css": "gulp scss:lint --prod",
     "lint:js": "gulp scripts:lint --prod",
     "compile": "babel -d dist src",
-    "test": "gulp scripts:test",
```

Comment:
No more `yarn test`?

Reply:
You can still run it with gulp scripts:test, the npm script is on;y really there as a helper for the prepare script above.

Reply:
I can add it back in for consistency?

Reply:
Is this removed because yarn test:cover does the same thing, or do we not want to run the tests on `prepare` now?

Reply:
Yeah `yarn test:cover` does the same thing, and also I imagine people are using the gulp scripts directly, right? The `test` script is kind of redundant here.

Reply:
Yeh, I just use the gulp scripts personally unless the yarn operation runs multiple things at once